vumi/message.py

```diff
@@ -210,6 +210,18 @@ class TransportUserMessage(TransportMessage):
     TRANSPORT_TYPES = set([TT_HTTP_API, TT_IRC, TT_TELNET, TT_TWITTER, TT_SMS,
                            TT_USSD, TT_XMPP, TT_MXIT, TT_WECHAT])
 
+    AT_IRC_NICKNAME = 'irc_nickname'
+    AT_TELNET_ADDRESS = 'telnet_address'
```

Comment:
I wasn't sure what the to and from addresses represent for telnet. IP address? Port?